mwl_phonemizer/crf_epitran_mwl.py

```
@@ -0,0 +1,255 @@
+        )
+        self.model.fit(X, y)
+
+        if self.crf_model_path:
+            self.save_model(self.crf_model_path)
+
+    def phonemize(self, word: str, lookup_word: bool = True) -> str:
+        word = word.lower().strip()
+        if lookup_word and word in self.GOLD:
+            return self.GOLD[word]
+        if not self.model:
+            raise ValueError("CRF model is not trained or loaded.")
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)